react/SlideToggle/SlideToggle.less

```diff
@@ -0,0 +1,82 @@
+@import (reference) "~seek-style-guide/theme";
+
+.root {
+  display: inline-flex;
```

Comment:
Not sure how well IE 11 plays with `inline-flex`. I'd give it a spin before merging.